packages/metadata-common/package.json

```diff
@@ -43,6 +43,7 @@
     "@lumino/messaging": "^1.4.3",
     "@lumino/widgets": "^1.19.0",
     "@material-ui/core": "^4.12.1",
+    "@rjsf/core": "^3.1.0",
```

Comment:
this is only added here but should also be in ui-components since it is referenced by the components as well. Though if we move the FormEditor to ui-components it would only need to be referenced there